config/application.rb

```diff
@@ -51,5 +51,7 @@ class Application < Rails::Application
     config.active_record.encryption.primary_key = ENV.fetch("ACTIVE_RECORD_ENCRYPTION__PRIMARY_KEY", nil)
     config.active_record.encryption.deterministic_key = ENV.fetch("ACTIVE_RECORD_ENCRYPTION__DETERMINISTIC_KEY", nil)
     config.active_record.encryption.key_derivation_salt = ENV.fetch("ACTIVE_RECORD_ENCRYPTION__KEY_DERIVATION_SALT", nil)
+
+    config.active_record.raise_on_assign_to_attr_readonly = false
```

Comment:
`config.load_defaults 7.1` causes attributes declared with `attr_readonly` such as here https://github.com/DFE-Digital/register-trainee-teachers/blob/f48f9e500b5308c50714f8249d5e9bd6ac41f465/app/models/concerns/sluggable.rb#L8 to raise. Opting out for now to be compatible with what we currently have as we'll need to change the behaviour of a few things if we want to enable this.